sound/soc/sof/control.c

```diff
@@ -392,8 +392,9 @@ int snd_sof_bytes_ext_get(struct snd_kcontrol *kcontrol,
 
 	/* Prevent read of other kernel data or possibly corrupt response */
 	data_size = cdata->data->size + sizeof(const struct sof_abi_hdr);
-	max_size = (size < max_size) ? size : max_size;
-	max_size = (be->max < max_size) ? be->max : max_size;
+	if (size < max_size) /* type warnings with min(), use test */
```

Comment:
here we are comparing with size(usigned int), so let's use type cast, or change to uint for max_size and be->max, to remove warning. 
And, we can use min3(a, b, c) to get that in one line.

Reply:
yes. done